frontend/packages/console-app/src/components/oauth-config/OAuthConfigDetailsPage.tsx

```diff
@@ -6,7 +6,8 @@ import { referenceForModel } from '@console/internal/module/k8s';
 import { OAuthConfigDetails } from './OAuthConfigDetails';
 
 const { common } = Kebab.factory;
-const menuActions = [...Kebab.getExtensionsActionsForKind(OAuthModel), ...common];
+// eslint-disable-next-line @typescript-eslint/no-non-null-assertion
```

Comment:
can we add some explanation for the eslint-disable in this line

```suggestion
// eslint-disable-next-line @typescript-eslint/no-non-null-assertion - some reason here
```

Reply:
fixed the type error